feature-gate/proto/feature-gate.proto

```diff
@@ -0,0 +1,58 @@
+syntax = "proto3";
```

Comment:
can we not use just the copy of this file in `/lib/ipc`?

Reply:
Lol. That actually is what happens currently. It just copies that into a local file for building and I accidentally pushed that.